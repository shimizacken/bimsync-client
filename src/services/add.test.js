import add from './add.js';

test('add 1 + 2 equils 3', () => {
    
    expect(add(1, 2)).toMatchSnapshot();
})