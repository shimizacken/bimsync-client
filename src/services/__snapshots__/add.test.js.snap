// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`add 1 + 2 equils 3 1`] = `3`;
